morphia/src/main/java/org/mongodb/morphia/annotations/Indexed.java

```diff
@@ -76,10 +76,7 @@
     boolean unique() default false;
 
     /**
-     * Indicates the direction of the index (ascending, descending); default is ascending
-     *
-     * @deprecated use the {@link IndexOptions} found in {@link #options()}
+     * Indicates the type of the index (ascending, descending); default is ascending
      */
-    @Deprecated
```

Comment:
because the other values are options to the index and this isn't.  i.e., there isn't a corresponding value in IndexOptions for this.  at the class level, this would defined in the @Field annotation.